__tests__/html/newMessageButton.tabOrder.html

```diff
@@ -63,7 +63,7 @@
 
```

Comment:
All pngs for tests appear to be size changes only, is that correct? (No visual changes)

Reply:
All PNGs have the button size changed, because we are no longer using `<button>` but `<li>`.